osgar/record.py

```diff
@@ -88,12 +88,16 @@ def main(record=record):
     parser.add_argument('--duration', help='recording duration (sec), default infinite', type=float)
     parser.add_argument('--log', help='force record log filename')
     parser.add_argument('--application', help='import string to application', default=None)
+    parser.add_argument('--param', action='append',
+                        help='optional configuration parameters like go.max_speed=0.1')
```

Comment:
What if more parameters need to be added? `nargs='*'` ?

Reply:
well for one `--param` there is only one value, which is "key=value" and not more arguments. You can specify multiple calls via `action='append'`

Reply:
example of yesterday run was `python -m osgar.record --application osgar.go:Go config/eduro.json --note "try old config" --param app.dist=0.2 --param app.timeout=10.0`

Reply:
I know. I would prefer just `--param app.dist=0.2 app.timeout=10.0`. There it could be also `--params`?

Reply:
OK, in that case it could be `--params` and nargs='+'

Reply:
OK, thanks m.